core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1030,6 +1032,16 @@ class HttpGoogleServicesDAO(
       .build()
   }
 
+  def getResourceBufferServiceAccountCredential: Credential = {
+    new GoogleCredential.Builder()
+      .setTransport(httpTransport)
+      .setJsonFactory(jsonFactory)
+      .setServiceAccountId(resourceBufferPemEmail)
+      .setServiceAccountScopes(Seq(ComputeScopes.CLOUD_PLATFORM).asJava)
```

Comment:
not entirely sure what scopes are needed here, but it errors if there are no scopes.